contentcuration/contentcuration/frontend/shared/views/ContentNodeLearningActivityIcon.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+
+  <span v-if="includeText">
+    <span v-for="activity in activities" :key="activity">
+      <div v-if="chip" :class="small ? 'small-chip' : 'chip'">
+        <KLabeledIcon :icon="icon(activity)" :label="text(activity)" />
+      </div>
+      <div v-else>
+        <KLabeledIcon :icon="icon(activity)" :label="text(activity)" />
+      </div>
+    </span>
+  </span>
+  <span v-else>
+    <span v-if="showEachActivityChip">
+      <KIcon
+        v-for="activity in activities"
+        :key="activity"
+        :icon="icon(activity)"
+        :aria-label="text(activity)"
+      />
+    </span>
+    <span v-else-if="activities.length > 1 && !showEachActivityChip">
+      <KIcon :icon="icon('multiple')" :aria-label="text('multiple')" />
+    </span>
+    <span v-else>
+      <KIcon :icon="icon(activities[0])" :aria-label="text(activities[0])" />
+    </span>
+  </span>
+
+</template>
+
+<script>
+
+  import { camelCase } from 'lodash';
+  import { LearningActivities } from '../constants';
+  import { getLearningActivityIcon } from 'shared/vuetify/icons';
+  import { metadataTranslationMixin } from 'shared/mixins';
+
+  export default {
+    name: 'ContentNodeLearningActivityIcon',
+    mixins: [metadataTranslationMixin],
+
+    props: {
+      learningActivity: {
```

Comment:
Not blocking, just would be nice. I think that `learningActivities` name for this prop would make a bit more sense because we can receive more than one. Also having a validator function would help to understand what values are expected here. The same applies to the same property in the `Thumbnail` component.